src/components/Cart/index.js

```diff
@@ -472,16 +466,6 @@ const CartUI = (props) => {
             />
           )}
         </CartSticky>
-        <Confirm
```

Comment:
@YisusJe  why are you deleting this component?

Reply:
Because was duplicate by accident

Reply:
@YisusJe It's ok this title ?
![image](https://user-images.githubusercontent.com/77166972/152818926-90b97517-f7a8-4164-a505-7e999d0198ba.png)

Reply:
let me change it for offers or products